test/test_loops.py

```diff
@@ -43,6 +43,21 @@ def test_for_inclusiverange_increment_two(self):
         input_js_code = 'for (let i = 0; i <= 10; i += 2) {\n\t5\n}'
         expected_py_code = 'for i in range (0, 12, 2):\n\t5'
         self.assertEqual(expected_py_code, main.main(input_js_code, 'js', 'py'))
+    
+    def test_for_with_update_expression_plus(self):
```

Comment:
Can we add some bash and java tests?

Reply:
We don't have for loops implemented in java and bash yet so can't test this use case there yet